SPECS/expat/expat.spec

```diff
@@ -27,7 +26,6 @@ This package contains minimal set of shared expat libraries.
 
 %prep
 %setup -q
-%patch0 -p1
 %build
 %configure \
```

Comment:
Nit: the `bindir` and `libdir` flags are already set by the `%configure` macro and are redundant here.